src/pages/User/User.jsx

```diff
@@ -88,31 +113,51 @@ const User = () => {
           }}
         >
           {!pollList.loading &&
-            pollList.data.map((dataList) => (
-              dataList.options.length >0 &&
-              <PollItem
-                title={dataList.title}
-                key={dataList._id}
-                InnerOption={dataList.options.map((option, i) => (
-                  <InnerPoll
-                    option={option.option}
-                    key={i}
-                    votes={
-                      <input
-                        type="radio"
-                        name={dataList._id}
-                        style={{ cursor: "pointer" }}
-                        onChange={() =>
-                          VoteChange(dataList.title, dataList._id, option.option)
+            paginatedPollList.map(
+              (dataList) =>
+                dataList.options.length > 0 && (
+                  <PollItem
+                    title={dataList.title}
+                    key={dataList._id}
+                    InnerOption={dataList.options.map((option, i) => (
+                      <InnerPoll
+                        option={option.option}
+                        key={i}
+                        votes={
+                          <input
+                            type="radio"
+                            name={dataList._id}
+                            style={{ cursor: "pointer" }}
+                            onChange={() =>
+                              VoteChange(
+                                dataList.title,
+                                dataList._id,
+                                option.option
+                              )
+                            }
+                            disabled={disabledOptions[dataList.title]}
+                          />
                         }
-                        disabled={disabledOptions[dataList.title]}
                       />
-                    }
+                    ))}
                   />
-                ))}
-              />
-            ))}
+                )
+            )}
         </Box>
+        <TablePagination
+          component="div"
+          rowsPerPageOptions={rowsPerPageOptions}
+          count={pollList.data.length}
+          page={
+            page >= Math.ceil(pollList.data.length / rowPerPage)
+              ? Math.max(0, Math.ceil(pollList.data.length / rowPerPage) - 1)
+              : page
```

Comment:
Can you separate this logic to a function that return actual value after calculations,
use that function here